### ISSUE TITLE
Check the default behavior for dialogs when a user clicks out of the dialog without making a selection.

### ISSUE THREAD

**Issue by ehamner** [CMU-CREATE-Lab/flutter-app-android#221]
Related to #220 

Should probably double check default behavior for any dialog.

**Comment by pdille** [comment]
This should be ongoing before every release, but @SteveFulton95 you willing to do a pass with the current state of the code?

**Comment by SteveFulton95** [comment]
Yeah I can do this. In the case of #220, I just accidentally forgot to remove a reference to a log that was on the flutter.  So the app thought the log that was selected just finished recording, and wanted you to save it to the kindle.  But that caused a crash because there was not actually a log on the flutter.

**Comment by SteveFulton95** [comment]
Thinking about this further, I am pretty sure the only dialogs that could potentially cause weird behavior on dismissal are the dialogs on the data logs tab.  Those dialogs are the only ones that we potentially want something to happen when a dialog is dismissed. Like resume reading data points until the recording has finished.

I'll go through and double check, though.

**Comment by SteveFulton95** [comment]
Found a crash in the clean up logs.  Forgot a null check.  Resolved in commit e1b1f9cca353a0d91d0c47daf5a74a9a6aa705fb.

Other than that I found nothing.  I am going to leave this open just in case we decide we want to do another run through before another release.

**Comment by pdille** [comment]
Awesome, thanks for going through it!

We'll be making a checklist of items to go over before every release and this will be one of them.
